server/config.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     server: get('DB_SERVER', 'localhost'),
     database: get('DB_NAME', 'licences'),
     sslEnabled: get('DB_SSL_ENABLED', 'true'),
+    port: 5432,
```

Comment:
Can this port number be overridden with an environment variable so that is could be deployed and configured in an environment where PG is running on a non-standard port?

Reply:
Done. Never has been, because it's dockerised so no reason for it not to be 5432